typings/config.d.ts

```diff
@@ -24,11 +24,11 @@ declare global {
         settings?: SettingsJson;
         passes?: PassJson[];
       }
-  
```

Comment:
someone's got a diff trailing space editor setting it seems :)

Reply:
ah, damn, eslint isn't checking .ts files, obvs. I'll find the setting :P